### ISSUE TITLE
make consistent module placing

### ISSUE THREAD

**Issue by ppanero** [inveniosoftware/invenio-communities#922]
invenio-records-resources defines a file/module hierarchy base on layers (records, services, resources). Then inside each one of them are the specifics for each _record type_ (e.g. communities, members, etc.). This module has a mix between an inverse approach defining root modules for record types (communities, members) and then inside each of them the layers. Additionally, there is a `records` module in the root which is confusing, a `schema.py` file in the root of communities which should be in the `services` layer, `permissions.py` in the root instead of in the `/services`...

TL;DR: needs mass file placement refactoring

```
/resources
|
|- /communities
|- /members 

/services
|
|- /communities
|- /members 

/records
|
|- /communities
|- /members 
```
